### ISSUE TITLE
Can provide reference wandb training log?

### ISSUE THREAD

**Issue by DylanLIiii** [huggingface/nanoVLM#76]
Currently I use default setting training the model on a 8xH20 GPUs. But the training log is strange. In the first eval, my model had an accuracy close to 0.3. Then it dropped to 0 in the next round and gradually increased. Can anyone provide a reference training log so that I can better understand what wrong is?

![Image](https://github.com/user-attachments/assets/48ab4b28-c8b4-4253-b0c7-533ad351766f)

**Comment by DylanLIiii** [comment]
> Hi, Did you use `torch.compile` ?

No, I didn't use torch.compile. I kept the default configuration from the repo, only adjusting the batch size to 32 per GPU to maintain a global batch size of 256 (guess it is global one).

**Comment by Jiajun-Chu** [comment]
Hi, I'm encountering the same issue. The accuracy started dropping from the beginning.

In my first two trials, I used the default settings, but I noticed that both batch_loss and val_loss increased unexpectedly. To address this, I applied gradient clipping in the next two trials, which smoothed out the loss curve. However, it didn't improve the accuracy.

![Image](https://github.com/user-attachments/assets/1f881aca-9fbe-4826-8095-848e32e4d895)

**Comment by pdh930105** [comment]
I have a similar issue. 

I tested 3 trial. (8 RTX3090 GPU 5 epoch (total BS=64*8), 8 RTX3090 15 epoch (total BS=64*8), 1 RTX3090 GPU 15 epoch (total BS=64))

In my assumption, DDP has breaked accuracy and overfitting.

![Image](https://github.com/user-attachments/assets/b01e5ca6-c4fa-4c41-a69f-9cb44b0b39c4)
![Image](https://github.com/user-attachments/assets/38d6271d-e598-452e-9656-db1f7ed903a1)
![Image](https://github.com/user-attachments/assets/b669d2f0-76fe-442e-ba18-c37af127ecf0)
